fw.lua

```diff
@@ -40,7 +40,7 @@ local function _equals(self, a, b)
 		end
 	else
 		_log(self, "Comparing " .. tostring(a) .. " and " .. tostring(b))
-		equals = a == b
+		equals = tostring(a) == tostring(b)
```

Comment:
Please keep commit atomic, this change seems unrelated to the context of this pull request.